blazor/datagrid/caption-template.md

```diff
@@ -0,0 +1,275 @@
+---
+layout: post
+title: Caption template in Blazor DataGrid Component | Syncfusion
+description: Checkout and learn here all about Grouping in Syncfusion Blazor DataGrid component and much more details.
+platform: Blazor
+control: DataGrid
+documentation: ug
+---
+
+# Caption template in Blazor DataGrid component
+
+The caption template feature in the Syncfusion Blazor DataGrid allows you to customize and enhance the appearance of group caption row. It provides a flexible way to display additional information about grouped data, such as counts or grouped value, and enables you to incorporate custom content like images, icons, or other HTML elements. This feature empowers you to create visually appealing and informative group captions in the DataGrid component.
+
+To achieve this customization, you can utilize the  [CaptionTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridGroupSettings.html#Syncfusion_Blazor_Grids_GridGroupSettings_CaptionTemplate) property. You can type cast the context as [CaptionTemplateContext](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.CaptionTemplateContext.html#properties) to get the data, which represents the currently displayed group, you can incorporate its properties such as `Field` (column's Field name), `HeaderText` (column's Header text), `Key`(grouped value) and `Count` (Count of the grouped records) into the Caption template.
+
+The following example demonstrates how to customize the group header caption in the Grid by utilizing  the `CaptionTemplate` property. It displays the **HeaderText**, **Key** and **Count** of the grouped columns.
+
+{% tabs %}
+{% highlight razor tabtitle="Index.razor" %}
+@using Syncfusion.Blazor.Grids
+
+<SfGrid  DataSource="@GridData" AllowGrouping="true" Height="315px">
+    <GridGroupSettings ShowDropArea="false" Columns="@Initial">
+        <CaptionTemplate>
+            @{
+                var data = (context as CaptionTemplateContext);
+                <span>@data.HeaderText-@data.Key : @data.Count Items </span>
```

Comment:
@data.HeaderText-@data.Key : @data.Count Items


leave space between data.HeaderText and data.Key

Reply:
Modified